copylot/hardware/prime_bsi_camera/camera.py

```diff
@@ -0,0 +1,182 @@
+from pyvcam import pvc, constants
+from pyvcam.camera import Camera
+
+
+class PrimeBSICamera:
+    def __init__(
+        self,
+        camera_name: str = None,
+        # TODO: create properties for these
```

Comment:
I think these shouldn't be properties. You mentioned you don't want to change these attributes often but more like you want to set it at the beginning only. That's why I intended to have them have them as arguments here instead of properties. What do you think @ieivanov ?

Reply:
Ah, I see your point. I think we should keep the `__init__` as simple as possible. We often don't know what are possible values for these properties and need to initialize the camera to find out. Also, we occasionally change them after we start up the camera - it would be inconvenient to have to unload the camera to change these properties.